test/Utils-spec.jsx

```diff
@@ -155,6 +156,13 @@ describe('Utils', () => {
         )).to.equal(false);
       });
 
+      it('should match children before and after interpolation', () => {
+        expect(nodeEqual(
+          <div>{2}{' children'}{<span />}</div>,
```

Comment:
can you make this line be `<div>{2}{' children'}{<span />} abc {'hey'}</div>,` so that we test that we work correctly with spaces irrespective of whether they're defined like `{'hey '}there`  or like `{'hey'} there`?